daml-lf/language/daml-lf.bzl

```diff
@@ -99,11 +86,11 @@ def _cmp(a, b):
         return -1
 
 def _cmp_lf_version(a, b):
-    if a == "1.dev" and b == "1.dev":
+    if (a == "1.dev" or a == "dev") and (b == "1.dev" or b == "dev"):
```

Comment:
this seems  weird, can we instead make the arguments we pass here consistently be `1.dev` instead of trying to accept both?

Reply:
Will do that in another PR.